src/webapp/components/collapsible-section/Section.tsx

```diff
@@ -0,0 +1,89 @@
+import { Icon, IconButton, makeStyles, Paper } from "@material-ui/core";
+import React from "react";
+
+type iconPosition = "left" | "right";
+type arrowStyle = "upDown" | "rightLeft";
+
+export interface SectionProps {
+    isOpen?: boolean;
+    setOpen?: (open: boolean) => void;
+    children: React.ReactNode;
+    title: React.ReactNode;
+    collapsible?: boolean;
+    elevation?: number;
+    iconPos?: iconPosition;
+    classProps?: {
+        section?: string;
+        header?: string;
+        content?: string;
+    };
+    arrowStyle?: arrowStyle;
+}
+
+export const Section = ({
+    children,
+    title,
+    collapsible,
+    isOpen = true,
+    setOpen = () => {},
+    elevation = 1,
+    iconPos = "right",
+    classProps = {},
+    arrowStyle = "upDown",
```

Comment:
About the expand/collapse styles, I checked what DHIS2 does, and... it depends:

orgUnits tree (before the name): right/down
maps (the icon being on the right): up/down

I have no strong opinion on this, so I'll left for the PM to decide.